ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -170,12 +170,12 @@ export default function BuildQuote({
   const showSmartTransactionsOptInPopover =
     smartTransactionsEnabled && !smartTransactionsOptInPopoverDisplayed;
 
-  const onCloseSmartTransactionsOptInPopover = (e) => {
+  const onManageStxInSettings = (e) => {
```

Comment:
In this case `onManageStxInSettings` is not the right name here, because the old UI (Build Quote component and others) don't have the transaction settings modal. However, we will be removing old, unused UI anyway (including this file), so it's ok.